src/CallWebhookJob.php

```diff
@@ -133,6 +141,7 @@ private function dispatchEvent(string $eventClass)
             $this->response,
             $this->errorType,
             $this->errorMessage,
+            $this->transferTime,
```

Comment:
This is a breaking change. Move the added option as the last one and make it optional.